### ISSUE TITLE
Support for other repo source (Gitlab, pull on hooks)

### ISSUE THREAD

**Issue by meteormanaged** [lipp/doclets#188]
Any plans or chance for support of other source other than github? I personally use gitlab (both a community server that I maintain, as well as sometimes using gitlab.com) and it'd be great to be able to gather from those sources.

If auth integration dev is a concern, perhaps a more simple approach could be used by providing an https url for a repo, as well as a username and password to access it, or a deploy key.

**Comment by lipp** [comment]
Yes, I'd like to support other repos too. No schedule when this will be ready though :(
